core/consensus/src/synchronization.rs

```diff
@@ -82,9 +97,25 @@ where
             self.check_proof(id, proof.clone())?;
 
             if id != 1 && current_hash != epoch.header.pre_hash {
+                trace::error(
+                    "prev_hash_diff".to_string(),
```

Comment:
The event is best described by an action, recommended to use `diff_prev_epoch_hash`